src/applications/vaos/services/appointment/index.js

```diff
@@ -542,6 +544,12 @@ export function groupAppointmentsByMonth(appointments) {
   return appointmentsByMonth;
 }
 
+export async function createAppointment({ appointment }) {
+  const result = await postAppointment(appointment);
+
+  return transformVAOSAppointment(result);
```

Comment:
When the VAOS service sends us the created appointment/request data, we probably want to transform that back into our Appointment format for use later. Though I don't think this is used for much currently.